source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -15,9 +15,36 @@ To import all the Sass rules from GOV.UK Frontend, add the following to your Sas
 @import "node_modules/govuk-frontend/govuk/all";
 ```
 
-### Import an individual component's CSS
+### Import specific parts of the CSS
 
-To import the [button](https://design-system.service.gov.uk/components/button/) component for example, add the following to your Sass file:
+If you want to improve how quickly Sass compiles to CSS, you can import only the Sass rules you need.
```

Comment:
This will also (and arguably a lot more importantly) improve performance for end users, as the compiled CSS will only include the bits required for the service, so it'll be smaller, and thus the page will load quicker.

Reply:
Ah thanks - updated in 148d922.